parm/soca/obs/config/obs_list.yaml

```diff
@@ -0,0 +1 @@
+../obs_list.yaml
```

Comment:
Not a fan of this relative path... but I guess it will be fine?

Reply:
My bad, I messed up in the g-w. I'll remove it in the future.